src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -557,6 +593,31 @@ public void MethodReturnParameterTest(string methodName, NullabilityState elemen
             Assert.Empty(nullability.GenericTypeArguments);
         }
 
+        public static IEnumerable<object[]> MethodReturnsTupleTestData()
+        {
+            // public Tuple<string?, string>? MethodReturnsTupleNullNonNull() => null;
+            yield return new object[] { "MethodReturnsTupleNullNonNull", NullabilityState.Nullable, NullabilityState.NotNull, NullabilityState.Nullable };
+            // public Tuple<string?, int> MethodReturnsTupleNullNonNot() => null!
+            yield return new object[] { "MethodReturnsTupleNullNonNot", NullabilityState.Nullable, NullabilityState.NotNull, NullabilityState.NotNull };
+            // public (int?, string)? MethodReturnsValueTupleNullNonNull() => null;
+            yield return new object[] { "MethodReturnsValueTupleNullNonNull", NullabilityState.Nullable, NullabilityState.NotNull, NullabilityState.Nullable };
+            // public (string?, string) MethodReturnsValueTupleNullNonNon() => (null, string.Empty);
+            yield return new object[] { "MethodReturnsValueTupleNullNonNon", NullabilityState.Nullable, NullabilityState.NotNull, NullabilityState.NotNull };
+        }
+
+        [Theory]
+        [SkipOnMono("Disabling NullablePublicOnly feature not work for Mono tests")]
```

Comment:
Should this be an ActiveIssue instead (with `PlatformDetection.MonoRuntime`)?

Reply:
There is no active issue, i believe the nullability attributes stay trimmed in mono, i think i need to change the message instead